src/client/public-openfin/rt-fx.json

```diff
@@ -13,6 +13,7 @@
     }
   },
   "runtime": {
+    "arguments": "--remote-debugging-port=9090",
```

Comment:
Connecting to openfin dev tools on 9090 is not working, all the applications are not served there except notifications (they are hidden), see [issue](https://bugs.chromium.org/p/chromium/issues/detail?id=1232509). I added that attribute to control the dev port in case I wanted to connect playwright with openfin application.

Reply:
still a bit confused about whether we should be settling on 9091 as the "standard" debug port for OpenFin, at least for the reactive ecosystem platform manifests (I realise the notifications manifest currently has 9090 burned into it)

Reply:
also, given we have added it to `rt-fx.json`, should it also be added as standard to all manifests (inc `rt-credit.json` and `launcher.json`)